packages/react/src/use-render/useRender.ts

```diff
@@ -12,25 +12,14 @@ export function useRender<
   State extends Record<string, unknown>,
   RenderedElementType extends Element,
 >(params: useRender.Parameters<State, RenderedElementType>): useRender.ReturnValue {
-  const { render, props, state, refs } = params;
-  const { ref: intrinsicRefProp, ...intrinsicProps } = props || {};
+  const renderParams = params as useRender.Parameters<State, RenderedElementType> & {
+    disableStyleHooks: boolean;
+  };
+  renderParams.disableStyleHooks = true;
 
-  const renderElement = useRenderElementLazy(
-    undefined,
-    { render },
-    {
-      props: intrinsicProps,
-      state,
-      ref: [intrinsicRefProp, ...(refs || [])].filter(
-        (x): x is React.Ref<RenderedElementType> => x != null,
-      ),
-      disableStyleHooks: true,
-    },
-  );
+  const element = useRenderElement(undefined, renderParams, renderParams);
 
-  return {
-    renderElement,
-  };
+  return element;
```

Comment:
I don't insist on the mutable `.disableStyleHooks` line, I just wrote it this way to highlight what this refactor made evident: `useRender` is really just `useRenderElement` but with style hooks turned off.

The prop/ref destructuring & handling logic was all unnecessary: it's duplicated with the logic in `useRenderElement` whic already does all that.